ui/src/datasource/DatasourceForm.vue

```diff
@@ -52,11 +52,12 @@
         :complete="dialogStep > 3"
         step="3"
       >
```

Comment:
Can we add a `v-if` that only shows this step if the schema service is reachable?